test/tesselation.jl

```diff
@@ -36,14 +36,20 @@
     mesh = tesselate(pset, VoronoiTesselation(StableRNG(2024)))
     @test crs(mesh) === crs(pset)
 
-    coords = [LatLon(rand(-90:T(0.1):90), rand(-180:T(0.1):180)) for _ in 1:10]
-    pset = PointSet(Point.(coords))
-    mesh = tesselate(pset, VoronoiTesselation(StableRNG(2024)))
-    @test crs(mesh) === crs(pset)
-
     # error: the number of coordinates of the points must be 2
     pts = randpoint3(10)
     pset = PointSet(pts)
     @test_throws AssertionError tesselate(pset, VoronoiTesselation(StableRNG(2024)))
+
+    # Test polygon order is the same as input points order
+    pts = randpoint2(10)
+    mesh = tesselate(pts, VoronoiTesselation(StableRNG(2024)))
+    @test all(zip(pts, mesh)) do (p, poly)
+      # check if the target polygon's centroid is the closest
+      dist(e) = evaluate(Euclidean(), p, centroid(e))
+      all(mesh) do elem
+        dist(elem) ≥ dist(poly)
+      end
+    end
```

Comment:
@disberd, define functions inside tests does not work in some cases.
Try move this test to a helper function like this:
```julia
# check if polygon order is the same as input points order
function issameorder(pts, mesh)
  all(zip(pts, mesh)) do (p, poly)
    # check if the target polygon's centroid is the closest
    dist(e) = evaluate(Euclidean(), p, centroid(e))
    all(dist(elem) ≥ dist(poly) for elem in mesh)
  end
end
```
Test helper functions lives in `test/testutils.jl`.
After this, use this function:
```julia
@test issameorder(pts, mesh)
```

Reply:
@disberd, I'll do this to help you with your PR.

Reply:
@eliascarv I introduced this function, so it is my fault. Please avoid adding it to testutils.jl, it is only going to be used here. Refactor the code without the function if necessary.